src/main/java/org/sagebionetworks/bridge/crypto/AesGcmEncryptor.java

```diff
@@ -67,7 +67,7 @@ private AesCipherService createCipher() {
         return cipher;
     }
 
-    private static final int KEY_BIT_SIZE = 256;
+    private static final int KEY_BIT_SIZE = 128;
```

Comment:
As far as I can tell, this is being used to encrypt the health code when we store participants in Redis. This could break something, and we don't have an easy way to test for this.

It's also quite concerning that upgrading Shiro forces us to downgrade our encryption. I'd reach out to Aaron H and ask him about this.

Reply:
This one is hairy. It could currently be ignoring the value and using the highest value it supports, in which case we'd be okay... but also it could not be doing that and we'd break everything. However, if we don't resolve this, we can't update our shiro library.

Reply:
I've removed this update for now, and filed a JIRA to track it: https://sagebionetworks.jira.com/browse/BRIDGE-2839